portfolio/src/main/webapp/script.js

```diff
@@ -190,4 +192,13 @@ function deleteComments(commentId) {
         const maxComments = document.getElementById('maxcomments').value;
         getComments(maxComments)
     })
+}
+
+/** Sets Filter on Comments */
+function setFilter() {
+    const maxComments = document.getElementById('maxcomments').value;
+    var filterInputNode = document.getElementById('filter-input');
+    const filterInputValue = filterInputNode.value.toLowerCase();
+    filterInputNode.setAttribute("data-filter-input", filterInputValue);
```

Comment:
Are you using this data attribute for anything?

Reply:
I was using it as a parameter for `getComments()` but I just realized I could've used the element's value instead. Will make some changes to this!